content/automate/ManageIQ/System/Notification/Email.class/infrastructuremiqprovisionrequestapproverapproved.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
@billfitzgerald0120 
Is there a state machine step that calls this instance, should that be attached to this PR?